client/src/featureform/register.py

```
@@ -216,6 +216,8 @@ def register_training_set(
         schedule: str = "",
         tags: Optional[List[str]] = None,
         properties: Optional[Dict] = None,
+        resource_snowflake_config: Optional[ResourceSnowflakeConfig] = None,
```

Ruff (0.8.2)

`ResourceSnowflakeConfig` may be undefined, or defined from star imports

(F405)